vc_assistance/setup.py

```diff
@@ -0,0 +1,35 @@
+# This file is part of the CERN Indico plugins.
+# Copyright (C) 2014 - 2018 CERN
+#
+# The CERN Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License; see
+# the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from setuptools import find_packages, setup
+
+
+setup(
+    name='indico-plugin-vc-assistance',
+    version='1.0',
```

Comment:
`3.0-dev` (our current versioning scheme uses 3.x for plugins requiring indico>=2.2)